doc/configuration.rst

```diff
@@ -99,6 +99,33 @@ Subclassing ``Config`` or ``DefaultConfig``
 
     Defaults to ``False``.
 
+  .. method:: cli_context(command: str) -> Iterator[None]
+
+    A context manager which wraps the execution of the CLI command. Use it to
```

Comment:
I'd like to give a bit more context here about _why_ MonkeyType might require this kind of setup/teardown. Maybe like this:

"A context manager which wraps the execution of a CLI command. MonkeyType has to import your code in order to generate stubs for it; in some cases (e.g. Django) some setup is required before code can be imported. Use this method to define the necessary setup or teardown for your codebase."